packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -853,7 +852,7 @@ class TimelineEvent {
   }
 
   void format(StringBuffer buf, String indent) {
-    buf.writeln('$indent$name [start: $startTime] [end: $endTime]');
+    buf.writeln('$indent$name ${timeRange.toString()}');
```

Comment:
here and elsewhere remove .toString() calls that don't do anything.

Reply:
done.